lib/mapping/MapEditUtils.cpp

```diff
@@ -235,8 +238,8 @@ CTerrainViewPatternConfig::CTerrainViewPatternConfig()
 					{
 						std::vector<std::string> range;
 						boost::split(range, mapping, boost::is_any_of("-"));
-						terGroupPattern.mapping.push_back(std::make_pair(boost::lexical_cast<int>(range[0]),
-							boost::lexical_cast<int>(range.size() > 1 ? range[1] : range[0])));
+						terGroupPattern.mapping.emplace_back(boost::lexical_cast<int>(range[0]),
```

Comment:
1) surprised that lexical_cast survived. Or is that only works for int -> string conversion? There is `std::stoi` that can be used here.
2) Something happened with formatting?

Reply:
1. Only for int -> string, yes. I will manually change it to std::stoi.

Reply:
Done with both.